systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -715,18 +726,28 @@ void testCertRenewalInMaintenanceWindow(ExtensionContext extensionContext) {
             .withNamespaceName(namespaceName)
             .withClusterName(clusterName)
             .withMessageCount(MESSAGE_COUNT)
+            .withSecurityProtocol(SecurityProtocol.SASL_SSL)
             .withKafkaUsername(user.getMetadata().getName())
             .withListenerName(Constants.TLS_LISTENER_DEFAULT_NAME)
             .build();
 
         Map<String, String> kafkaPods = PodUtils.podSnapshot(namespaceName, kafkaSelector);
 
-        LOGGER.info("Annotate secret {} with secret force-renew annotation", secretName);
-        Secret secret = new SecretBuilder(kubeClient(namespaceName).getSecret(namespaceName, secretName))
+        LOGGER.info("Annotate secret {} with secret force-renew annotation", clusterSecretName);
+        Secret secretCa = new SecretBuilder(kubeClient(namespaceName).getSecret(namespaceName, clusterSecretName))
             .editMetadata()
                 .addToAnnotations(Ca.ANNO_STRIMZI_IO_FORCE_RENEW, "true")
             .endMetadata().build();
-        kubeClient(namespaceName).patchSecret(namespaceName, secretName, secret);
+        assertThat("Cluster CA certificate has not been regenerated yet", secretCa.getMetadata().getAnnotations().get("strimzi.io/ca-cert-generation"), is("0"));
+        kubeClient(namespaceName).patchSecret(namespaceName, clusterSecretName, secretCa);
+
+        LOGGER.info("Annotate secret {} with secret force-renew annotation", clusterSecretName);
+        Secret secretCli = new SecretBuilder(kubeClient(namespaceName).getSecret(namespaceName, clientsSecretName))
+            .editMetadata()
+                .addToAnnotations(Ca.ANNO_STRIMZI_IO_FORCE_RENEW, "true")
+            .endMetadata().build();
+        assertThat("Clients CA certificate has not been regenerated yet", secretCli.getMetadata().getAnnotations().get("strimzi.io/ca-cert-generation"), is("0"));
```

Comment:
```suggestion
        assertThat("Clients CA certificate has not been regenerated yet", secretCli.getMetadata().getAnnotations().get(Ca.ANNO_STRIMZI_IO_CA_CERT_GENERATION), is("0"));
```
applies for others as well